tests/ovirt/basic_vm_import_negative_test.go

```diff
@@ -85,7 +85,7 @@ var _ = Describe("VM import", func() {
 		created, err := f.VMImportClient.V2vV1alpha1().VirtualMachineImports(namespace).Create(&vmi)
 
 		Expect(err).NotTo(HaveOccurred())
-		Expect(created).To(HaveValidationFailure(f, string(v2vv1alpha1.SecretNotFound)))
+		Expect(created).To(BeUnsuccessful(f, string(v2vv1alpha1.ValidationFailed)))
```

Comment:
@jakub-dzon I ended up just with `BeUnsuccessful` instead maintaining the previous assertion `HaveValidationFailure` since updating the status.condition isn't deterministic. In order not to block this PR, I'm using `BeUnsuccessful`, but we should explore why the status.conditions is being overridden in certain cases only by the Successful condition type, and in other cases the Valid condition type remain as well.

Reply:
Could you, please, create an issue to track that?

Reply:
Done - https://github.com/kubevirt/vm-import-operator/issues/322